irods/manager/data_object_manager.py

```diff
@@ -974,6 +975,7 @@ def modDataObjMeta(self, data_obj_info, meta_dict, **options):
                 in_pdmo="",
                 next=0,
                 rescId=0,
+                dataAccessTime=""
```

Comment:
does this deserve a trailing comma too?

Reply:
Yes probably,and we'll run it through black too.